### PR TITLE
gofmt and govet all of test-infra

### PR DESCRIPTION
**Pull request by spxtr**
This isn't the ideal way to do this, since there's a bit of wasted work. We rerun gofmt and go vet for every file, not just the ones that have changed.

Still, they're pretty quick. gofmt takes <1 second and go vet takes ~4 seconds on my machine.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,11 +35,9 @@ script:
     - pylint queue-health/weekly_commit_stats.py  # TODO(fejta): all python files
     - ./jenkins/diff-job-config-patch.sh
     - ./jenkins/diff-e2e-runner.sh
-    - make -C gcsweb test
```
Comment (fejta) [inline .travis.yml]: Is this intentional? I assume the idea here was to ensure this builds?

File: .travis.yml [hunk 2]
```diff
@@ -35,11 +35,9 @@ script:
...
     - cd gubernator
     - ./test.sh --nologcapture
     - ./lint.sh
     - ../node_modules/.bin/mocha static/build_test.js
     - cd ..
-    - chmod +x testgrid/jenkins_verify/verify.sh
```
Comment (fejta) [inline .travis.yml]: Is this already +x now?

File: .travis.yml (hunk 1 above)
Comment (spxtr) [inline .travis.yml]: It already builds and tests with bazel. The only thing it was doing previously was gofmt and govet, but now that's unnecessary.

File: .travis.yml (hunk 2 above)
Comment (spxtr) [inline .travis.yml]: I set it in this PR.

File: verify/verify-gofmt.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2016 The Kubernetes Authors.
```
Comment (ixdy) [inline verify/verify-gofmt.sh]: 2017? :)

File: verify/verify-gofmt.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Prefer gofmt from io_bazel_rules_go_toolchain.
```
Comment (ixdy) [inline verify/verify-gofmt.sh]: bazel doesn't set a PATH? :(

Comment (spxtr) [inline verify/verify-gofmt.sh]: Nope :(

File: verify/verify-gofmt.sh (hunk 3 above)
Comment (spxtr) [inline verify/verify-gofmt.sh]: bah!

**Pull request merged**